README.md

```diff
@@ -889,8 +895,55 @@ Similarly, when running tests in PyCharm, its own proprietary test discovery
 process may also increase the chance of the `AzulTestCase` context manager
 causing a failure.
 
-If these failures occur, add the warning to the list of permitted warnings found
-in [`AzulTestCase`](test/azul_test_case.py) and commit the modifications. 
+If these failures occur, add the warning to the list of permitted warnings
+found in [`AzulTestCase`](test/azul_test_case.py) and commit the modifications. 
+
+
+## Installing Python 3.8.3 on macOS BigSur
+
+[pyenv Big Sur GitHub issue](https://github.com/pyenv/pyenv/issues/1740)
+
+Users of macOS 11 or later may encounter a `build failed` error when installing
+Python through pyenv. A patch was made available to remedy this:
+
+First, ensure that bzip2 and any other requirements for the Python build
+environment are met.
+See [pyenv wiki](https://github.com/pyenv/pyenv/wiki#suggested-build-environment)
```

Comment:
Empty line between paragraphs, but this shouldn't be a new paragraph or even a new line. To avoid having the link run over the margin, use reference-style links.